### PR TITLE
Oar huge

### PR DESCRIPTION
**Pull request by curoli**

### CODE REVIEW

File: huge-cache/src/main/scala/org/broadinstitute/dig/aggregator/methods/hugecache/HugeCacheStage.scala [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  val variantCqs: Input.Source   = Input.Source.Success(variantCqsDir)
+  val nearestGenes: Input.Source = Input.Source.Success(nearestGenesDir)
+
+  /** Source inputs. */
+  override val sources: Seq[Input.Source] = Seq(variantCqs, nearestGenes)
+
+  /* Define settings for the cluster to run the job.
+   */
+  override val cluster: ClusterDef = {
+    super.cluster.copy(instances = 10)
```
Comment (psmadbec) [inline huge-cache/src/main/scala/org/broadinstitute/dig/aggregator/methods/hugecache/HugeCacheStage.scala:35]: Is 10 instances strictly necessary here? How long does it take to run with a more normal number like 5?

File: huge-common/src/main/scala/org/broadinstitute/dig/aggregator/methods/huge/HugeCommonStage.scala [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      releaseLabel = ReleaseLabel("emr-6.7.0")
+    )
+  }
+
+  /** Map inputs to outputs. */
+  override val rules: PartialFunction[Input, Outputs] = {
+    case genes()                        => Outputs.All
+    case geneAssociations(_, phenotype) => Outputs.Named(phenotype)
+    case variants(phenotype)            => Outputs.Named(phenotype)
+    case cache()                        => Outputs.All
```
Comment (psmadbec) [inline huge-common/src/main/scala/org/broadinstitute/dig/aggregator/methods/huge/HugeCommonStage.scala:55]: This is basically mean that we always run all phenotypes whenever a new dataset is added. I would remove the `cache()` bit entirely (although I would keep the genes one). The logic is non-ideal, but this is it in short:

VEP runs whenever a dataset is added, but all SNPs from a dataset will be in VEP (and the csq won't change for a SNP between runs, it is all offline). So the actual data for a phenotype can't change unless a dataset is added for that phenotype. So you don't need to run all phenotypes when the cache runs, you only need to run the phenotypes that have changed.

**Comment by curoli** [comment]
Hello Patrick, 
I'm not sure what you mean by you want "look more like the huge-rare (e.g. huge-rare/src/main/resources/huge-rare.py) rather than huge-common (e.g. huge-common/src/main/scala/org/broadinstitute/dig/aggregator/methods/huge/HugeCommonStage.scala)". Are you saying you want directories org/broadinstitute/dig/aggregator/methods/huge removed? I know in Scala directories are not required to follow package structure, but usually, they do.
Best, Oliver

**Comment by psmadbec** [comment]
I'm saying that the rest of the aggregator methods have a specific file structure:

`bottom-line/src/main/scala/BottomLine.scala`

And I prefer if new methods match that structure. If you generate a new project via `sbt new broadinstitute/dig-aggregator-method.g8` (https://github.com/broadinstitute/dig-aggregator-method.g8) it will use that structure as well.

**Comment by curoli** [comment]
OK, I think I did all you requested.

**Review by psmadbec — APPROVED** [review]
Yeah LGTM

**Pull request merged**